### ISSUE TITLE
Does not give new version available suggestions

### ISSUE THREAD

**Issue by krishnalalstha** [handstandsam/AndroidDependencyManagement#6]
I successfully implemented but when there is new version, does not give new version available suggestions

**Comment by tbsandee** [comment]
This almost surely is an issue with Android Studio.

**Comment by krishnalalstha** [comment]
But it gives when all the dependency defined in the build.gradle file

**Comment by tbsandee** [comment]
Yes, you're right, it's true. But it's a bug in lint or Android Studio. There's no reason lint shouldn't be able to walk through the dependencies when they come from a buildSrc file.

**Comment by kaeptmblaubaer1000** [comment]
@tbsandee have you created a bug report (in Google's bug tracker)?

**Comment by tbsandee** [comment]
Nope, I've been meaning to file several issues related to using dependencies from buildSrc kt files, but haven't had time yet.

**Comment by XabierGoros** [comment]
Yeap, same thoughts here. I like pretty much the solution, but cannot make it work with Android Studio and the dependencies new version lint. Also, I've tested with Android Studio 3.3, gradle plugin 3.3.0-alpha06 and wrapper 4.9 and the Kotlin navigation doesn't work for this very project.

**Comment by handstandsam** [comment]
Here is a workaround from @hitherejoe to keep track of newer versions of libraries, but it only works when you run it in CI, not in Android Studio though.

He commented on my caster.io lesson on it: https://caster.io/lessons/gradle-dependency-management-using-kotlin-and-buildsrc-for-buildgradle-autocomplete-in-android-studio
![screen shot 2019-01-13 at 3 29 33 pm](https://user-images.githubusercontent.com/264948/51090318-37a2a180-1748-11e9-9135-ea5f4567f953.png)

Article: https://overflow.buffer.com/2018/10/26/keeping-check-on-your-android-libraries-with-danger-bitrise/

**Comment by heitorcolangelo** [comment]
Do you guys know the issue link on Google issue tracker?
